webcompat/issues.py

```diff
@@ -13,10 +13,34 @@
 
 from flask import abort
 
+from webcompat import app
 from webcompat import github
 from webcompat.form import build_formdata
 from webcompat.helpers import proxy_request
-from webcompat.helpers import REPO_URI
+
+REPO_URI = app.config['ISSUES_REPO_URI']
+PRIVATE_REPO_URI = app.config['PRIVATE_REPO_URI']
+
+
+def unmoderated_issue():
+    """Gets the placeholder data to send for unmoderated issues."""
+    # TODO: Replace this with something meaningful.
+    # See https://github.com/webcompat/webcompat.com/issues/3137
+    summary = 'Placeholder in-moderation title.'
+    body = 'Placeholder in-moderation body.'
+    return {'title': summary, 'body': body}
+
+
+def report_private_issue(form, public_url):
+    """Report the anonymous users issue to our private repo.
+
+    This also allows us to pass in public_url metadata, to be
+    embedded in the issue body.
+    """
+    path = 'repos/{0}'.format(PRIVATE_REPO_URI)
+    formdata = build_formdata(form, ('public_url', public_url))
```

Comment:
hmm… not sure the code should extend the logic here. Probably there should be another function for adding stuff to the form object, which is just a dictionary.

So if there was a 

```python
form = add_metadata(form, {'key': 'value'})
```

or even more directly:

```python
form['public_url'] = public_url
```

then the modification to `build_formdata` is minimal.

```diff
-    metadata_keys = ['browser', 'ua_header', 'reported_with']
+    metadata_keys = ['browser', 'ua_header', 'reported_with', 'public_url']
```

Reply:
I'll try that approach, I agree this part felt a bit gross after it evolved.

Reply:
I've implemented the direct solution, but I might create the add_metadata way in a bit -- it seems useful perhaps. Thanks for the suggestions, I was missing the obvious simple solution. :)))